packages/frontend/src/components/UserSettings/ProfilePanel/index.tsx

```diff
@@ -25,6 +18,7 @@ import {
 import { VerifyEmailModal } from '../../../pages/VerifyEmail';
 import { useApp } from '../../../providers/AppProvider';
 import { useErrorLogs } from '../../../providers/ErrorLogsProvider';
+import MantineIcon from '../../common/MantineIcon';
 
 const updateUserQuery = async (data: Partial<UpdateUserArgs>) =>
```

Comment:
out of scope but if you you move this api request and mutation hook to `packages/frontend/src/hooks/user/useUser.ts` that'll be grand

Reply:
yes yes, I have that in my mind to change after!